include/blmc_robots/n_joint_blmc_robot_driver.hxx

```diff
@@ -417,7 +417,13 @@ bool NJBRD::homing(NJBRD::Vector endstop_search_torques_Nm,
                                       home_offset_rad,
                                       index_search_step_sizes);
 
-    rt_printf("Finished homing.\n");
+    rt_printf("Finished homing.  Offset: ");
```

Comment:
i was wondering whether we could have a somewhat more informative print. i think for people who don't know what it is, this will be confusing.

Reply:
I agree, I'm just not sure what would be less confusing without being unnecessarily verbose.
Ideally I would like to have an option to enable/disable the print as it is only meaningful at all for the case of home offset calibration when the robot was manually moved to zero before starting the homing.  The problem is, that the driver is not accessible from the user code, so right now there is no good way to pass this option down to the driver.